src/server.js

```diff
@@ -40,6 +42,10 @@ async function start(model) {
         bot.on('message', Commands.MessageHandler)
         //TODO: handle sending files
         bot.launch().then(() => console.log(bot._name + " started")).catch(console.error)
+
+        // Enable graceful stop
+        process.once('SIGINT', () => bot.stop('SIGINT'));
```

Comment:
Could separate this out into another PR, cos this would be good in main branch but we may not want to include the tangles specific stuff